posthog/models/feature_flag.py

```diff
@@ -124,6 +125,10 @@ def variants(self):
                 return variants
         return []
 
+    @property
+    def variants_by_key(self):
```

Comment:
payloads can exist on a non-variant flag as well, so this seems like the wrong place for the payload to be in?

Reply:
addressed here #13623